### DISCUSSION TITLE
What is the capital of 🇵🇪?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#972] [Galaxy Trivia] [answered]
**Bonus:** Guess the iconic river, vegetable, and animal of the country.

**Comment by peterbaccaro** [comment]
The capital of Peru is **Lima**. Lima is located on the central coast of the country, overlooking the Pacific Ocean. It is the largest city in Peru and serves as the political, cultural, and economic centre of the country.

The iconic river of Peru is the **Amazon River**. It is the largest river in terms of volume and one of the longest rivers in the world. The Amazon River begins in the Peruvian Andes and flows through several countries, including Peru, before reaching the Atlantic Ocean. It is known for its immense biodiversity and plays a significant role in the Amazon rainforest ecosystem. The Amazon River and its surrounding rainforest are considered a natural treasure and one of the most ecologically diverse regions on Earth.

The iconic vegetable of Peru is the **potato**. Peru is known as the birthplace and centre of origin for potatoes. The country boasts an incredible variety of potato cultivars, with over 4,000 different types grown across its diverse regions. Potatoes have been a staple food in Peru for thousands of years, and they hold great cultural, historical, and culinary significance in Peruvian cuisine. The country's rich potato heritage and the various ways they are prepared and enjoyed make them an iconic vegetable in Peru.

The iconic animal of Peru is the **vicuña**. Related to the llama, the vicuña is a small graceful camelid that is native to the Andes Mountains of South America, including Peru. It is highly prized for its incredibly soft and fine wool, which is used to produce luxurious textiles. The vicuña is considered a symbol of the Andean highlands and is protected as a national treasure in Peru.

**Comment by Sriansh-raj** [comment, marked as answer]
> **Bonus:** Guess the iconic river, vegetable, and animal of the country.

The capital of Peru is **Lima**.
 
--> Some of the iconic symbols of Peru include:
**River**: The most iconic river in Peru is the **Amazon River**, which is the second-longest river in the world and runs through several countries in South America, including Peru.
 

**Vegetable**: One of the most famous vegetables in Peru is the **potato**. Peru is actually the birthplace of the potato, and there are over 4,000 different varieties of potatoes grown in the country
.

**Animal**: The national animal of Peru is the **vicuña**, which is a type of South American camelid that is related to llamas and alpacas
.

In addition to these symbols, Peru is also known for its rich history and culture, as well as its delicious cuisine. Lima, the capital city of Peru, has been transformed into the culinary capital of South America, and is known for its ceviche, a popular seafood dish
.
